playbooks/files/rax-maas/plugins/managed_k8s_services_local_check.py

```diff
@@ -68,11 +68,11 @@ def check(auth_ref, args):
         milliseconds = (dt.microseconds + dt.seconds * 10 ** 6) / 10 ** 3
 
     status_ok(m_name=service_name)
-    metric_bool('mk8s_{service}_local_status'.format(service=args.service),
+    metric_bool('managed_k8s_{service}_local_status'.format(service=args.service),
```

Comment:
This line expands to long so pep8 check failed:

/var/lib/jenkins/workspace/PR_rpc-maas-master-container-pep8-tox-test/rpc-maas/playbooks/files/rax-maas/plugins/managed_k8s_services_local_check.py:71:80: E501 line too long (82 > 79 characters)

Need to wrap line around